src/features/auth/pages/login/Login.native.test.tsx

```diff
@@ -320,19 +324,88 @@ describe('<Login/>', () => {
 
     expect(analytics.logSignUp).toHaveBeenNthCalledWith(1, { from: 'Login' })
   })
+
+  describe('Login from offer favorite modal', () => {
+    const OFFER_ID = addFavoriteJsonResponseSnap.offer.id
+
+    it('should redirect to Offer page WHEN signin is successful and user comes from offer', async () => {
```

Comment:
POURQUOI WHEN EST EN MAJUSCULE ?

```suggestion
    it('should redirect to Offer page when signin is successful and user comes from offer', async () => {
```

Reply:
Parce que c'est comme ça dans tous les tests de ce fichier 😆

Reply:
c'est comme ça dans 12 fichiers seulement

la majorité des fichiers c'est `when`

je trouve ça bizarre de le mettre en majuscule